analytic_engine/src/sst/factory.rs

```diff
@@ -169,9 +148,10 @@ impl Factory for FactoryImpl {
         options: &SstReaderOptions,
         path: &'a Path,
         store_picker: &'a ObjectStorePickerRef,
+        file_size: u64,
```

Comment:
How about defining a struct(for example, `FileContext`?) to wrap `path` and `file_size` ?

Reply:
Can't think of any advantage to wrap them together.